app/english/vocabulary/page.tsx

```
@@ -1,7 +1,16 @@
-import { FadeIn } from "@/components/animations/fade-in"
-import { VocabularyPractice } from "@/components/english/vocabulary-practice"
+import { FadeIn } from "@/components/animations/fade-in";
+import { VocabularyPractice } from "@/components/English/vocabulary-practice";
+import { createClient } from "@/lib/supabase/server";
+import next from "next";
```

ESLint

[error] 'next' is defined but never used.

(@typescript-eslint/no-unused-vars)